sdks/wasm/test.js

```diff
@@ -147,6 +119,24 @@ var send_message = find_method (driver_class, "Send", -1)
 if (!send_message)
 	throw 3;
 
+//Ok, this is temporary
+//this is a super big hack (move it to a decently named assembly, at the very least)
+var binding_test_module = assembly_load ("binding_tests");
+
+//binding test suite support code
+var binding_test_class = find_class (binding_test_module, "", "TestClass");
+if (!binding_test_class)
+	throw 9;
+
+function call_test_method(method_name, signature, args)
+{
+	var target_method = find_method (binding_test_class, method_name, -1)
```

Comment:
Why -1? To get rid of magic numbers, could we make this a `#define FIND_METHOD_UNLIMITED` or something?

Reply:
Good point, I'll change the code to use a constant.